plugin/discovery/get.go

```diff
@@ -298,7 +298,7 @@ func (i *ProviderInstaller) install(provider string, version Version, url string
 		// check if the target dir exists, and create it if not
 		var err error
 		if _, StatErr := os.Stat(i.Dir); os.IsNotExist(StatErr) {
-			err = os.Mkdir(i.Dir, 0700)
+			err = os.MkdirAll(i.Dir, 0700)
```

Comment:
This is likely the fix for the other issues regarding directory permissions during init too.